Makefile

```diff
@@ -219,6 +235,13 @@ device-trees: $(SOURCES_RESTRICTED) $(DESTDIR)/boot-assets
 		-name "*.dtbo" -o -name "overlay_map.dtb") \
 		$(DESTDIR)/boot-assets/overlays/
 
+kernel-initrd: $(SOURCES_MAIN)
+	# Update the kernel version in extra_content.yaml
+	$(eval KERNEL_VERSION := $(shell apt-cache $(APT_OPTIONS_KERNEL) policy linux-raspi | grep Candidate | cut -d " " -f 4 | cut -d "." -f 1-4))
```

Comment:
This makes an assumption about the kernel being installed, which is part of the image definition, no? Do gadget paths support globs?

Reply:
Gadget paths do not support globs unfortunately. This does require coordination between ubuntu-image and this gadget, but since the gadget tree is now being built as part of the ubuntu-image state machine, it should be a bit safer.